docker/oms-events/Dockerfile.dev

```diff
@@ -9,17 +9,17 @@ RUN mkdir -p /usr/src/shared
 #WORKDIR /usr/src/app
 
 RUN mkdir -p /usr/app/ \
-        && mkdir -p /usr/app/frontend \
         && mkdir -p /usr/app/media \
         && mkdir -p /usr/scripts \
-        && mkdir -p /usr/shared
+        && mkdir -p /usr/shared \
+        && mkdir -p /usr/app/oms-events
 
-WORKDIR /usr/app
+WORKDIR /usr/app/oms-events
```

Comment:
Can we use `/usr/app/src` for it, as in this article https://oms-project.atlassian.net/wiki/spaces/GENERAL/pages/24219843/Standardized+folder+namings?

Reply:
Agree, however I did not want to change everything at once to keep diff clarity. If you would prefer changing this at the same time that is fine with me.

Reply:
I actually think it would do no harm to implement it in this PR

Reply:
@i3anaan I'll try to update the folders to be standardized today in the evening, shouldn't be a big deal.